content/zh/docs/reference/glossary/name.md

```diff
@@ -16,7 +16,7 @@ tags:
 title: Name
 id: name
 date: 2018-04-12
-full_link: /zh/docs/concepts/overview/working-with-objects/names/
+full_link: /docs/concepts/overview/working-with-objects/names/
```

Comment:
Why this change?

Reply:
It is the original English version, commented out.

Reply:
> Why this change?

It is the original English version, commented out.